src/Mvc/Extensions.ApiDescription.Server/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -17,11 +23,27 @@
       DependsOnTargets="GenerateOpenApiDocuments" />
 
   <Target Name="OpenApiGetDocuments" Returns="@(_OpenApiProjectDocuments)">
+    <ItemGroup>
+      <_Temporary Remove="@(_Temporary)" />
+      <_Temporary Include="$(TargetFrameworks)" Exclude="netcoreapp1.0;netcoreapp1.1;netcoreapp2.0" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_Temporary>@(_Temporary)</_Temporary>
+    </PropertyGroup>
+
     <MSBuild Projects="$(MSBuildProjectFile)"
         Targets="OpenApiGetDocuments"
-        Properties="TargetFramework=$(TargetFrameworks.Split(';')[0])"
+        Condition=" '$(_Temporary)' != '' "
+        Properties="TargetFramework=$(_Temporary.Split(';')[0])"
         RemoveProperties="RuntimeIdentifier">
       <Output TaskParameter="TargetOutputs" ItemName="_OpenApiProjectDocuments" />
     </MSBuild>
+
+    <ItemGroup>
+      <_Temporary Remove="@(_Temporary)" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_Temporary />
+    </PropertyGroup>
```

Comment:
I'm not totally sure that all of this messing temporaries is really required. There are property functions to check if an item group is empty and such.

Reply:
Not checking anything. This is just about removing properties and items that other targets may use i.e. clean up. Leaving as is because someone on the MSBuild team complimented me on this practice 😺